### PR TITLE
add `action_group` and `check_mode` attributes

### LINKED ISSUE #197: consider adding "attributes" to collection content
##### SUMMARY

Attributes show up in documentation, see: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_role_module.html#attributes

Attribs: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/action_common_attributes.py#L11

Use in docs fragments will crash `ansible-doc` on older versions of Ansible 2.9/-base/-core.

For plugins, could crash Asnible itself on those older versions (unsure), however, attributes do not appear to be very useful in plugins as almost all of them apply only to modules/actions.

However, collection-specific attribs may be possible, and those would become useful in plugins too; we could define various attributes related to Vault functionality perhaps.

It seems there's little documentation on this for now.

##### ISSUE TYPE
- Feature Idea

### PR DESCRIPTION
**Pull request by briantist**
##### SUMMARY

Resolves #197 

##### ISSUE TYPE

- Docs Pull Request
- Feature Pull Request

### CODE REVIEW

File: plugins/doc_fragments/action_group.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class ModuleDocFragment(object):
+
+    DOCUMENTATION = r'''
+options: {}
+attributes:
+  action_group:
+    description: Use C(group/community.hashi_vault.vault) in C(module_defaults) to set defaults for this module.
+    support: full
+    membership:
+      - vault
```
Comment (felixfontein) [inline plugins/doc_fragments/action_group.py:20]: I wonder whether this should be
```suggestion
      - community.hashi_vault.vault
```
After all, top-level action groups and other collection's action groups can extend action groups for this collection, so always using the long form is probably best.

**Comment by felixfontein** [comment]
https://github.com/ansible-community/antsibull-docs/pull/62 will fix rendering this.

**Comment by felixfontein** [comment]
The antsibull-docs PR is merged and antsibull-docs 1.7.3 released.

**Comment by felixfontein** [comment]
Looks much better now: https://ansible-collections.github.io/community.hashi_vault/pr/320/collections/community/hashi_vault/vault_kv1_get_module.html#attributes

**Review by felixfontein — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]

**Comment by briantist** [comment]
@felixfontein thanks for reviewing and putting in that fix to `antsibull-docs` so quickly!

**Pull request merged**